stdlib/public/runtime/ErrorObject.mm

```diff
@@ -153,16 +167,27 @@ - (BOOL)isEqual:(id)other {
 @end
 
 Class swift::getNSErrorClass() {
-  return SWIFT_LAZY_CONSTANT([NSError class]);
+  return SWIFT_LAZY_CONSTANT(objc_lookUpClass("NSError"));
 }
 
 const Metadata *swift::getNSErrorMetadata() {
   return SWIFT_LAZY_CONSTANT(
     swift_getObjCClassMetadata((const ClassMetadata *)getNSErrorClass()));
 }
 
+static Class getAndBridgeSwiftNativeNSErrorClass() {
+  Class nsErrorClass = swift::getNSErrorClass();
+  assert(nsErrorClass && "Attempted to bridge an Error without Foundation");
```

Comment:
This seems pretty suspect. `as AnyObject` should do *something* when Foundation isn't loaded.

Reply:
This is definitely one of the WIP-y bits of the latest push. I may need to just leave it with the dummy superclass.